server/issue.go

```diff
@@ -185,6 +185,13 @@ func httpAPIGetCreateIssueMetadata(ji Instance, w http.ResponseWriter, r *http.R
 	}
 
 	w.Header().Set("Content-Type", "application/json")
+
+	if len(cimd.Projects) == 0 {
+		fmt.Fprintf(w, "{\"error\": \"You do not have permission to create issues in any projects. Please contact your Jira admin.\"}")
+		return http.StatusInternalServerError,
+			errors.WithMessage(err, "failed to marshal response")
```

Comment:
Copy paste error: `"failed to marshal response"`

Reply:
@jfrerich @crspeller I went ahead to fix the cut&paste but then noticed that as far as I can tell we should be returning a 200 OK here, not a 500, so I changed that too. Don't have an easy way of testing it though.

Reply:
Thanks @levb. I had to research the usage of writing directly with Fprintf vs saving to var bb of type []byte.